test/e2e/bound_sa_token_test.go

```diff
@@ -226,6 +227,8 @@ func TestTokenRequestAndReview(t *testing.T) {
 // TestChangeServiceAccountIssuer checks that the operator considers
 // the value set for Authentication.ServiceAccountIssuer when setting
 // the configuration configmap for the apiserver pods.
+//
+// Note: this test will roll out a new version
 func TestChangeServiceAccountIssuer(t *testing.T) {
```

Comment:
@marun  It seems that this test simply sets and check if a flag `apiServerArguments/service-account-issuer` appears in the config map. It will also trigger a deployment twice. So it would take ~20 minutes to complete. What's the real value of this test?